### ISSUE TITLE
Firebase.json does not work for multi-word ios app name

### ISSUE THREAD

**Issue by msadura** [invertase/react-native-firebase#3148]
## Issue
I installed firerbase core module correctly and integrated with firebase console. After adding admob module my ios app crashhes on startup. Android works fine.
I added `firebase.json` file with admob's ids but according to xcode error admob id is not set:
`'GADInvalidInitializationException', reason: 'The Google Mobile Ads SDK was initialized without an application ID`

I looked into xcode's build logs, and I found such log for RNFB build script:
```
info: -> RNFB build script started
info: 1) Locating firebase.json file:
info:      (1 of 2) Searching in '/Users/mar/projects/game-slide' for a firebase.json file.
info:      firebase.json found at /Users/mar/projects/game-slide/firebase.json
info: 2) Injecting Info.plist entries: 
    ->  0) firebase_json_raw string eyJhZG1vYl9pb3NfYXBwX2lkIjogImNhLWFwcC1wdWItMzAwMjUxMzA5NDQ2NTAxMn4zNTI2ODg3MDUxIiwgImFkbW9iX2FuZHJvaWRfYXBwX2lkIjogImNhLWFwcC1wdWItMzAwMjUxMzA5NDQ2NTAxMn43ODU3MjA3ODIzIn0=
    ->  1) firebase_crashlytics_collection_enabled bool NO
    ->  2) GADApplicationIdentifier string ca-app-pub-xxx-hidden-id
warning:   A Info.plist build output file was not found (/Users/mar/Library/Developer/Xcode/DerivedData/testing-fktgmmfdkecpredkbnokhugprapq/Build/Products/Debug-iphoneos/Space)
warning:   A Info.plist build output file was not found (Riddle.app/Info.plist)
warning:   A Info.plist build output file was not found (/Users/mar/Library/Developer/Xcode/DerivedData/GameSlide-fktgmmfdkecpredkbnokhugprapq/Build/Products/Debug-iphoneos/Space)
warning:   A Info.plist build output file was not found (Riddle.app.dSYM/Contents/Info.plist)
info: <- RNFB build script finished
```

As far as I understand it fails to localize Info.plist to inject values from firebase.json. Logs look pretty weird and I realised it is because my app has a space in product name - Space Riddle
After changing app name to single word it works.

---

### Javascript

<details><summary>Click To Expand</summary>
<p>

#### `package.json`:

```json
# N/A
```

#### `firebase.json` for react-native-firebase v6:

```json
{
  "react-native": {
    "admob_android_app_id": "ca-app-pub-3002513094465012~7857207823",
    "admob_ios_app_id": "ca-app-pub-3002513094465012~3526887051"
  }
}
```

### iOS

<details><summary>Click To Expand</summary>
<p>

#### `ios/Podfile`:

- [ ] I'm not using Pods
- [x] I'm using Pods and my Podfile looks like:

```ruby
# N/A
```

#### `AppDelegate.m`:

```objc
// N/A
```

</p>
</details>

---

### Android

<details><summary>Click To Expand</summary>
<p>

#### Have you converted to AndroidX?

- [ ] my application is an AndroidX application?
- [ ] I am using `android/gradle.settings` `jetifier=true` for Android compatibility?
- [ ] I am using the NPM package `jetifier` for react-native compatibility?

#### `android/build.gradle`:

```groovy
// N/A
```

#### `android/app/build.gradle`:

```groovy
// N/A
```

#### `android/settings.gradle`:

```groovy
// N/A
```

#### `MainApplication.java`:

```java
// N/A
```

#### `AndroidManifest.xml`:

```xml
<!-- N/A -->
```

</p>
</details>

---

## Environment

<details><summary>Click To Expand</summary>
<p>

**`react-native info` output:**

```
System:
    OS: macOS 10.14.6
    CPU: (8) x64 Intel(R) Core(TM) i7-6920HQ CPU @ 2.90GHz
    Memory: 135.61 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 11.15.0 - ~/.nvm/versions/node/v11.15.0/bin/node
    Yarn: 1.17.3 - /usr/local/bin/yarn
    npm: 6.7.0 - ~/.nvm/versions/node/v11.15.0/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
    Android SDK:
      API Levels: 23, 26, 27, 28
      Build Tools: 27.0.3, 28.0.2, 28.0.3, 29.0.0
      System Images: android-27 | Intel x86 Atom_64, android-27 | Google APIs Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Google Play Intel x86 Atom
  IDEs:
    Android Studio: 3.4 AI-183.6156.11.34.5522156
    Xcode: 11.3/11C29 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.5 => 0.61.5
```

- **Platform that you're experiencing the issue on**:
  - [ x] iOS
  - [ ] Android
  - [ ] **iOS** but have not tested behavior on Android
  - [ ] **Android** but have not tested behavior on iOS
  - [ ] Both
- **`react-native-firebase` version you're using that has this issue:**
  - `6.2`
- **`Firebase` module(s) you're using that has the issue:**
  - `e.g. Instance ID`
- **Are you using `TypeScript`?**
  - `Y/N` & `VERSION`
  
</p>
</details>

---

Think `react-native-firebase` is great? Please consider supporting all of the project maintainers and contributors by donating via our [Open Collective](https://opencollective.com/react-native-firebase/donate) where all contributors can submit expenses. [[Learn More]](https://invertase.io/oss/react-native-firebase/contributing/donations-expenses)

- 👉 Check out [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter for updates on the library.

**Comment by Salakar** [comment]
Hey @msadura - I believe @twelve17 is working on a fix for this in https://github.com/invertase/react-native-firebase/pull/3135

**Comment by twelve17** [comment]
@msadura  see https://github.com/invertase/react-native-firebase/pull/3152

**Comment by msadura** [comment]
Awesome news! I was searching for admob related issues first and somehow I missed that PR. 
Thank you, I will check it out as soon as it is available :) 💪

**Comment by Salakar** [comment]
This fix will be live in 6.3.0 this week, thanks
